android/src/main/java/nordpol/ui/NfcGuideView.java

```diff
@@ -70,6 +72,8 @@
 
     public NfcGuideView(Context context, AttributeSet attrs) {
         super(context, attrs);
+        
+        guideItemsStartDistance = getResources().getInteger(R.integer.GUIDE_ITEMS_START_DISTANCE);
```

Comment:
"guideItemsStartDistance" is now saved as a resource to make Android automatically choose the version most fitting for the current screen resolution. This makes the distance between the two devices in the NfcGuideView more alike no matter what screen resolution you have.